core/modules/qproc/QuerySession.cc

```diff
@@ -379,20 +379,9 @@ std::vector<std::string> QuerySession::_buildChunkQueries(query::QueryTemplate::
     }
     qana::QueryMapping const& queryMapping = *_context->queryMapping;
 
-    if (!queryMapping.hasSubChunks()) { // Non-subchunked
-        for(auto tupleIter=queryTemplates.begin(), e=queryTemplates.end(); tupleIter != e; ++tupleIter) {
-            std::string str = _context->queryMapping->apply(chunkSpec, *tupleIter);
-            chunkQueries.push_back(str);
-        }
-    } else { // subchunked:
-        ChunkSpecSingle::Vector sVector = ChunkSpecSingle::makeVector(chunkSpec);
-        for(auto& chunkStr : sVector) {
-            for(auto& qTemplate : queryTemplates) {
-                std::string str = _context->queryMapping->apply(chunkStr, qTemplate);
-                LOGS(_log, LOG_LVL_DEBUG, "adding query " << str);
-                chunkQueries.push_back(str);
-            }
-        }
```

Comment:
since we don't create subchunk query strings in the master anymore, the `else` part is no longer necessary.